offchain-modules/cli/src/commands/mod.rs

```diff
@@ -1,13 +1,13 @@
 pub mod types;
-use anyhow::{Error, Result};
+use anyhow::Result;
 use ethabi::Token;
+use force_eth_lib::relay::ckb_relay::CKBRelayer;
 use force_eth_lib::transfer::to_ckb::{approve, get_header_rlp, lock_eth, lock_token};
-use force_eth_lib::transfer::to_eth::burn;
+use force_eth_lib::transfer::to_eth::{burn, parse_ckb_proof};
+use force_eth_lib::util::eth_util::convert_eth_address;
 use log::debug;
 use types::*;
```

Comment:
do not use `::*`, import them explicitly.

Reply:
all the data structure which is the command Args defined in the `type module` will be used in the commands.
It will be so many items if  import them explicitly. So `types::*` could be better

Reply:
> do not use `::*`, import them explicitly.

all the data structure which is the command Args defined in the type module will be used in the commands.
It will be so many items if import them explicitly. So types::* could be better